### ISSUE TITLE
Remove llvm-strip pass from Makefile

### ISSUE THREAD

**Issue by qmonnet** [libbpf/bpftool#161]
> [...] bpftool expects llvm-strip (for no good reason, anymore), so make sure you have that tool available.
> 
> [...] it would be nice to remove llvm-strip, `bpftool gen object` strips out DWARF already, so you shouldn't need extra stripping.

_Originally posted by @anakryiko in https://github.com/libbpf/libbpf-bootstrap/issues/286#issuecomment-2339664076_

**Comment by chentao-kernel** [comment]
HI, qmonnet, you mean remove llvm-strip like this?
![image](https://github.com/user-attachments/assets/9fd5970f-c851-4c64-9e0f-9d828f7ad552)

**Comment by chentao-kernel** [comment]
It seems no one fix it in upstrem, i can send a ptach latter, do you think it's okay?

**Comment by qmonnet** [comment]
Absolutely. I planned to do it but I've been rather busy lately, so I won't do it this month. Go ahead, by all means!

**Comment by qmonnet** [comment]
https://lore.kernel.org/all/20240924165202.1379930-1-chen.dylane@gmail.com/t/#u

**Comment by qmonnet** [comment]
It's been a while, but my understanding from the discussion after the patch linked above was submitted is that we needed the call to llvm-strip after all. Closing the issue.
